projects/pipelines/roles/pipelines_run_kfp_notebook/tasks/main.yml

```diff
@@ -197,7 +197,7 @@
   - name: Create the secret token
     shell:
       oc create secret generic "{{ secret_token_name }}"
-         "--from-literal=token=$(oc create token '{{ sa_name }}' -n '{{ notebook_namespace }}')"
+         "--from-literal=token=$(oc create token '{{ sa_name }}' -n '{{ notebook_namespace }}' --duration=21600s)"
```

Comment:
I don't see where this is defined? 🤔 
I wanted to check the meaning of `duration`
https://kubernetes.io/docs/reference/kubectl/generated/kubectl_create/kubectl_create_secret_generic/

Reply:
I still don't understand 🤔 
It's a parameter of `oc create token`, fine ... but you're calling `oc create secret` 🤔 🤔

Reply:
Sorry I don't understand what the confusion is? The full command is being called is:
```
oc create token '{{ sa_name }}' -n '{{ notebook_namespace }}' --duration=21600s
```
This generates the token that is passed to `oc create secret generic ...`.

Reply:
oh I see, I didn't notice `oc create token` was being called in a `$(....)` 🙃